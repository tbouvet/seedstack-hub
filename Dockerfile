FROM java:alpine

ENV HUB_VERSION=1.0.0

RUN mkdir /opt
ADD https://oss.jfrog.org/artifactory/oss-snapshot-local/org/seedstack/hub/seedstack-hub/${HUB_VERSION}-SNAPSHOT/seedstack-hub-${HUB_VERSION}-20160408.190101-20-capsule.jar /opt/capsule.jar

EXPOSE 8080

CMD exec java $JAVA_OPTS -jar /opt/capsule.jar

