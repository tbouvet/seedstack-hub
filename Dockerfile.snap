FROM java:alpine
RUN apk update && apk add wget

ENV HUB_VERSION=1.0.0-SNAPSHOT

RUN mkdir /scripts
COPY scripts/install.sh /scripts/install.sh 
RUN chmod +x /scripts/install.sh 
RUN /scripts/install.sh

EXPOSE 8080

CMD exec java $JAVA_OPTS -jar /opt/capsule.jar
